public/src/app/applications/find-panel/find-panel.component.ts

```diff
@@ -37,14 +35,13 @@ export class FindPanelComponent implements OnDestroy, OnInit {
   
   public filterHash: string;
   private ngUnsubscribe: Subject<void> = new Subject<void>();
-  public workflowState: _.Dictionary<WorkflowStateCode>;
   private fomFilters: Map<string, IFilter | IMultiFilter>;
   public fomNumberFilter = new Filter<number>({ filter: { queryParam: 'fomNumber', value: null }});
   public forestClientNameFilter = new Filter<string>({ filter: { queryParam: 'fcName', value: null }});
   public commentStatusFilters: MultiFilter<boolean>; // For 'Commenting Open' or 'Commenting Closed'.
   public postedOnAfterFilter = new Filter<Date>({ filter: { queryParam: 'pdOnAfter', value: null } });
-  readonly minDate = moment('2018-03-23').toDate(); // first app created
-  readonly maxDate = moment().toDate(); // today
+  readonly minDate = DateTime.fromISO('2018-03-23').toJSDate(); // first app created
```

Comment:
This date is not correct (too far in the past), so I question whathever this logic is for...

Reply:
Hi @basilv, This date was not changed this time, but I did not know why this date was set as such. Seems it was already like this when FOM was copying over from previous project... Do you want to change this? But I don't see it will actually affect anything, it is a minimum date in the past for datePicker you can pick for "postedOnAfter".